proto/src/lib.rs

```diff
@@ -265,13 +265,16 @@ pub mod runtime {
 
         // Wait for the spawned process to open the endpoint port.
         // Connecting instantly does not give it enough time.
-        let channel = tokio::time::timeout(Duration::from_millis(500), async move {
+        let channel = tokio::time::timeout(Duration::from_millis(7000), async move {
```

Comment:
nit: Why do we need to change this given you've introduced the exponential backoff?

Reply:
The discord thread saw a case where the startup took -> 510 <- ms 😅. So I increased it a lot to be sure.

Reply:
Got it. It is more than a 10x increase. My thinking is that the exponential backoff is more than enough to handle any variance. The 500 vs 510 vs 7000 would be handled very nicely by it. I wouldn't change the initial 500ms we're waiting for.

Reply:
500/7000 is the maximum time for the entire backoff to happen within. This means several attempts will be made if the connect() call fails instantly. It seemed to me like the windows connect call took much longer (perhaps all of those 510 ms?) which means we want a looser max time.
The backoff is mainly for preventing log spam in the case of failure.

Reply:
Oh, I misread this. Yeh, keeping the backoff within the 7000 ms should be good. Resolving this.